docker_test.go

```diff
@@ -1836,7 +1836,7 @@ func TestContainerCapAdd(t *testing.T) {
 func TestContainerRunningCheckingStatusCode(t *testing.T) {
 	ctx := context.Background()
 	req := ContainerRequest{
-		Image:        "influxdb:1.8.10-alpine",
+		Image:        "influxdb:1.8.10", // influxdb doesn't provide an alpine+arm build (https://github.com/influxdata/influxdata-docker/issues/335)
 		ExposedPorts: []string{"8086/tcp"},
```

Comment:
I think this change is fine, and want to ask you about using the `ImagePlatform` field from the container request:
```suggestion
		Image:         "influxdb:1.8.10-alpine",
		ExposedPorts:  []string{"8086/tcp"},
		ImagePlatform: "linux/amd64", // influxdb doesn't provide an alpine+arm build (https://github.com/influxdata/influxdata-docker/issues/335)
```

The image size is practically irrelevant in bytes, although the non-alpine weights twice:

```table
influxdb                     1.8.10                     ba9488c478c5   7 days ago      295MB
influxdb                     1.8.10-alpine              3ab4fc5517d7   7 weeks ago     164MB
```


Whatever you prefer is fine.

Reply:
Hmmm, I'll give it a shot. I forogt you can run amd64 on Mac Docker Desktop .

Reply:
Confirmed, this runs fine. I'm just happy the test passes now, so whichever you prefer.

Reply:
Cool, I think it's more clear with the image platform being explicit. I'll approve it as is, if I may